charts/core/templates/controller-deployment.yaml

```diff
@@ -199,7 +206,7 @@ apiVersion: policy/v1beta1
 kind: PodDisruptionBudget
 metadata:
   name: neuvector-controller-pdb
-  namespace: neuvector
```

Comment:
This piece has been modified. Can you remove it?

Reply:
I reverted the change as desired. Still I do not understand, how the PDB should be working, if the helm-chart is installed in any other namespace than 'neuvector'